v1/ao/layer.go

```diff
@@ -343,6 +362,19 @@ func (s *span) GetTransactionName() string {
 	return s.aoCtx.GetTransactionName()
 }
 
+func (s *span) produceKVsFromAllSources(opts SpanOptions, args ...interface{}) []interface{} {
+		s.GetTrace().AddGlobalTags(opts.GlobalTags)
```

Comment:
Should this be a part of the `addTagsFromOpts` so that it handles all the tags operations for any opts value? (ie both backtrace and global tags)